test/cbmc/include/core_mqtt_config.h

```diff
@@ -70,4 +70,18 @@ struct NetworkContext
  */
 #define MQTT_PINGRESP_TIMEOUT_MS                ( 500U )
 
+/**
+ * @brief The maximum duration of receiving no data over network when
+ * attempting to read an incoming MQTT packet by the #MQTT_ProcessLoop or
+ * #MQTT_ReceiveLoop API functions.
+ *
+ * When an incoming MQTT packet is detected, the transport receive function
+ * may be called multiple times until all the expected number of bytes for the
+ * packet are received. This timeout represents the maximum duration of polling
+ * for any data to be received over the network for the incoming.
+ * If the timeout expires, the #MQTT_ProcessLoop or #MQTT_ReceiveLoop functions
+ * return #MQTTRecvFailed.
+ */
+#define MQTT_RECV_POLLING_TIMEOUT_MS            ( 3U )
```

Comment:
Try changing this to 2
```suggestion
#define MQTT_RECV_POLLING_TIMEOUT_MS            ( 2U )
```